inc/class-wpseo-utils.php

```diff
@@ -1097,4 +1097,21 @@ public static function is_metabox_active( $identifier, $type ) {
 	public static function is_woocommerce_active() {
 		return class_exists( 'Woocommerce' );
 	}
+
+	/**
+	 * Getter for the Adminl10n array. Applies the wpseo_admin_l10n filter.
+	 *
+	 * @return array The Adminl10n array.
+	 */
+	public static function getAdminL10n() {
```

Comment:
CS, this function should be in snake case format, for example: `get_admin_l10n`
This also needs to be changed in 
-class-taxonomy.php 
-class-metabox.php